syntaxes/templ.tmLanguage.json

```diff
@@ -389,41 +386,53 @@
         }
       ]
     },
-    "empty-import-expression":{
-      "match": "@(?:[A-z_][A-z_0-9]*\\.)?[A-z_][A-z_0-9]*\\(.*\\)\\s*$",
-      "captures": {
-        "0": {
+    "import-expression": {
+      "patterns": [
+        {
+          "name": "start.import-expression.templ",
```

Comment:
This first pattern captures from `@` to the opening parenthesis and uses the existing `import-expression-start`. Then the content inside the parenthesis is syntax marked by `source.go`.